socorro/cron/crontabber_app.py

```diff
@@ -952,6 +909,8 @@ def _run_one(self, job_class, config, force=False):
         last_success = None
         now = utc_now()
         log_run = True
+
+        exc_type = exc_value = exc_tb = None
```

Comment:
This needed to get hoisted up because if you hit ctrl-c, then it errors out because these names don't exist in that flow.